client-js/app/elements/labrad-plot.ts

```diff
@@ -1,9 +1,17 @@
 import 'd3';
-
+import * as three from 'three';
 import {viridisData} from '../scripts/colormaps';
 import * as datavault from "../scripts/datavault";
 
+/**
+ * Import the THREE namespace from the module.
+ */
+const THREE = three.default;
```

Comment:
Yes, I did track down that I could probably fix it by changing the typings file, but I didn't like the idea of changing it manually. Doing this will lead to problems whenever someone tries to update the typings in the future without knowing to update the format of the exports.